src/docker/start.sh

```diff
@@ -188,6 +188,8 @@ else
   sleep 60 && unregister &
   # Run the agent
   bash run-docker.sh "$@" --once &
+else
+  bash run-docker.sh "$@" &
```

Comment:
If once is removed, agent will be reused by the Azure DevOps server. But, KEDA will still try to scale it, thus rendering a second. This update doesn't seem sufficient.

Furthermore, in case of a Kubernetes pod failure, it won't be restarted, and the service will be lost (see `restartPolicy` to `Never`).